### ISSUE TITLE
Accessing ash from docker UI DSM6.0

### ISSUE THREAD

**Issue by dogboy23** [drgroot/docker-containers#23]
Hi there,

There was a custom command I used to use to access ash from the DSM, but I can't seem to get it to work it makes the page hang... I thought it was /bin/sh but maybe you can correct me [and put a line in the readme if there is a working command?]

Thank you again!

**Comment by drgroot** [comment]
No your correct, it should be /bin/sh

```
root@host:~# docker exec -it rtorrent /bin/sh                             
/ # which sh                                                                 
/bin/sh   
```

I will update the readme!

**Comment by dogboy23** [comment]
this doesn't work from within the DSM ui unfortunately anymore, it definitely used to...

**Comment by drgroot** [comment]
hmm, okay i will take a look tonight.

**Comment by dogboy23** [comment]
This is now resolved on updating the DSM [6.1] and reinstalling the docker container, so it's all OK now... unknown reason for fix
